server/middleware/error-handler.js

```diff
@@ -1,29 +1,37 @@
 /* eslint no-unused-vars: 0 */
 /* eslint consistent-return: 0 */
-
 import validator from 'express-validation';
 
+/**
+ * Handles and customizes incoming errors
+ * @param {object} err Validation errors from joi and express validation
+ * @param {object} req Request object
+ * @param {object} res Response object
+ * @param {Function} next Next middleware in the chain
+ */
```

Comment:
Add the `@return` value for this function, and all other functions where you defined a JSDoc without it.